src/python/pants/backend/codegen/utils.py

```diff
@@ -24,39 +24,64 @@ def find_python_runtime_library_or_raise_error(
     codegen_address: Address,
     runtime_library_module: str,
     *,
+    resolve: str,
+    resolves_enabled: bool,
     recommended_requirement_name: str,
     recommended_requirement_url: str,
     disable_inference_option: str,
 ) -> Address:
     addresses = [
         module_provider.addr
         for module_provider in module_mapping.providers_for_module(
-            runtime_library_module, resolve=None
+            runtime_library_module, resolve=resolve
```

Comment:
Reminder that if `--no-enable-resolves`, Pants will have set the resolve name to `<ignore-me>` or something like that. So everything ends up being in the same resolve.